content/03-reference/01-tools-and-interfaces/05-prisma-cli/02-command-reference.mdx

```diff
@@ -467,6 +467,99 @@ model Profile {
 
 </CodeWithResult>
 
+## <inlinecode>db</inlinecode>
+
+### <inlinecode>db push</inlinecode>
+
+> **Note**: This feature is currently in [Preview](../../../more/releases#preview). Features in Preview may still have some issues, and are not recommended for use in production environments. To access commands in Preview, you must opt-in via the `--preview-feature` flag.
+
+The `db push` command pushes the state of your Prisma schema file to the database without using migrations. This command is a good choice when you do not need to version schema changes, such as during prototyping and local development. `db push` works as follows:
+
+- `db push` introspects the database, and infers and executes the changes required to make your database schema reflect the state of your Prisma schema.
+- By default, after changes have been applied to the database schema, generators are triggered (for example, Prisma Client). You do not need to manually invoke `prisma generate`.
+- If `db push` anticipates that the changes could result in data loss, it will:
+  
+  - Throw an error 
+  - Require the `—-force` option if you still want to make the changes
+
+- `db push` does not interact with or rely on migrations. The migrations table will no be updated, and no migration files will be generated. If migrations are detected, `db  push` will prompt for confirmation or require an explicit option (`--ignore-migrations`) 
```

Comment:
POssibly worth explaining why the explicit option is useful.

Reply:
I'll link to further down in the docs where it's explained

Reply:
The option supresses the interactive prompt to confirm,.